tests/fps_tests.py

```diff
@@ -704,6 +738,155 @@ def test_expected_esld(self):
         self.assertEqual(loaded_sets, expected_sets)
         self.assertEqual(fp.error_list, [])
 
+class TestFindDiff(unittest.TestCase):
+    def test_unchanged_sets(self):
+        old_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com",
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        new_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com",
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        diff_sets, subtracted_sets = find_diff_sets(old_sets, new_sets)
+        self.assertEqual(diff_sets, {})
+        self.assertEqual(subtracted_sets, {})
+
+    def test_added_set(self):
+        old_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com",
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        new_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com",
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    ),
+            'https://primary2.com': 
+            FpsSet(
+                    primary="https://primary2.com",
+                    ccTLDs={
+                        "https://primary2.com": ["https://primary2.ca"]
+                    }
+                    )
+        }
+        expected_diff_sets = {
+            'https://primary2.com': 
+            FpsSet(
+                    primary="https://primary2.com",
+                    ccTLDs={
+                        "https://primary2.com": ["https://primary2.ca"]
+                    }
+                    )
+        }
+        diff_sets, subtracted_sets = find_diff_sets(old_sets, new_sets)
+        self.assertEqual(diff_sets, expected_diff_sets)
+        self.assertEqual(subtracted_sets, {})
```

Comment:
```suggestion
        self.assertEqual(find_diff_sets(old_sets, new_sets), (expected_diff_sets, {}))
```

Optional: you could test both return values at once, if you wanted to make this a bit more concise.